{"name": "test_error_app_key", "status": "passed", "description": "错误的app_key参数", "start": 1599986296167, "stop": 1599986296187, "uuid": "c7ad364c-059f-48e1-9900-cbae5e78d5db", "historyId": "c65a9a8f558881df192a38af12688d4e", "testCaseId": "b02efde331dcd4e27dafc6ea28e8f72c", "fullName": "case.test_login_pytest.Test_Login_user#test_error_app_key", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login_pytest"}, {"name": "subSuite", "value": "Test_Login_user"}, {"name": "host", "value": "V_LFJJLIU-NB0"}, {"name": "thread", "value": "19516-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login_pytest"}]}